server/services/interventionsService.test.ts

```diff
@@ -1886,11 +1875,6 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
               {
```

Comment:
_nit_ this `activities` section is irrelevant anyway from the point of "PATCH request to set the number of sessions on the action plan" right? might be good to review later to cut off expectations that are not relevant

Reply:
I suppose so, I guess I quite like having the whole response in the test so it's clear on our side exactly what we'll get, but maybe I should just be checking the swagger documentation for that?